metadata.json

```diff
@@ -83,7 +83,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 5.5.8 < 7.0.0"
+      "version_requirement": ">= 6.0.0 < 8.0.0"
```

Comment:
https://github.com/voxpupuli/voxpupuli.github.io/issues/229 is still open, but I think the minimum version should be 6.1.0 since that doesn't require `systemctl daemon-reload` which is nice.